src/server/config_registry.h

```diff
@@ -14,17 +14,32 @@ class ConfigRegistry {
   // Accepts the new value as argument. Return true if config was successfully updated.
   using WriteCb = std::function<bool(const absl::CommandLineFlag&)>;
 
-  ConfigRegistry& Register(std::string_view name, WriteCb cb);
-  ConfigRegistry& Register(std::string_view name);
+  ConfigRegistry& Register(std::string_view name, bool is_mutable, WriteCb cb);
+  ConfigRegistry& Register(std::string_view name, bool is_mutable);
```

Comment:
How about having a `Register(name)` method, and `RegisterMutable(name, cb)` versions? (cb can default to empty)
It's cleaner on the callsite, and slightly here as well I think

Reply:
Good idea. Fixed.